_posts/2021-04-21-unstructured-knowledge-sharing.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: epic
+title: "Unstructured Knowledge Sharing"
+date: 2021-04-21
+categories: [teams, culture, people]
+author: steve-hicks
+---
+
+- Ash wrote about _structured_ knowledge shares - /blog/2020/12/09/share-your-knowledge/
+  - but we do some good _unstructured_ knowledge sharing. Not scheduled, ad hoc.
+  - probably link somewhere to knowledge sharing as a core principle in some docs somewhere?
+- The dev-help slack channel - cc @damassi https://twitter.com/ArtsyOpenSource/status/1357819867638812672
+  - a place to ask technical questions when you're stuck
+  - we've built a large knowledge-base
+  - link to auto-solved bot
+  - sometimes it's not a question - it's something we already figured out but thought it was worth sharing (contribution to the knowledge base)
```

Comment:
@zephraph has good examples of this in slack: https://artsy.slack.com/archives/C9ZUJE65V/p1617987884001700